eth_account/account.py

```diff
@@ -614,7 +635,9 @@ def signHash(self, message_hash, private_key):
         return self._sign_hash(message_hash, private_key)
 
     @combomethod
-    def _sign_hash(self, message_hash, private_key):
+    def _sign_hash(self,
+                   message_hash: Hash32,
+                   private_key: Union[bytes, HexStr, int, keys.PrivateKey]) -> Any:
```

Comment:
Any i/o SignedMessage because of decorator

Reply:
same here:
```suggestion
                   private_key: Union[bytes, HexStr, int, keys.PrivateKey]) -> SignedMessage:
```